docs/source/cli_help/traverse.rst

```diff
@@ -27,6 +27,10 @@ Traverses the branch tree in pre-order (i.e. simply in the order as they occur i
 By default ``traverse`` starts from the current branch.
 This behaviour can, however, be customized using options: ``--start-from=``, ``--whole`` or ``-w``, ``-W``.
 
+By default ``traverse`` asks if the branch should be pushed, this behaviour can, however, be changed with the ``machete.traverse.push`` configuration key.
```

Comment:
I'd move it below, to right before `If the traverse flow is stopped..`

Reply:
Or maybe even better, as a Note within `if the branch is not tracked on a remote...` bullet point?